src/component/Footer/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoFilter } from '../../types/TodoFilter';
+
+interface FooterProps {
+  todos: Todo[]
```

Comment:
Consider changing the type of 'filteredBy' from 'string' to 'TodoFilter' for better type safety.